src/core/crypto_tls.c

```diff
@@ -1074,6 +1081,19 @@ QuicCryptoTlsEncodeTransportParameters(
             Connection,
             "TP: Disable 1-RTT Encryption");
     }
+    if (TransportParams->Flags & QUIC_TP_FLAG_VERSION_NEGOTIATION) {
+        TPBuf =
+            TlsWriteTransportParam(
+                QUIC_TP_ID_VERSION_NEGOTIATION_EXT,
+                (uint16_t)TransportParams->VersionNegotiationInfoLength,
+                TransportParams->VersionNegotiationInfo,
+                TPBuf);
+        QuicTraceLogConnVerbose(
```

Comment:
I think we should add more encode/decode tracing for the full TP details.

Reply:
I need to sync up with @ThadHouse on how to add a new decoder to CLOG, because it would be nice to log the version lists, at least.

Reply:
Added some more tracing in the VNI TP decoding logic in version_neg.c.